tools/schemacode/schemacode/render.py

```diff
@@ -539,7 +379,7 @@ def make_entity_table(schema, tablefmt="github", **kwargs):
     def _remove_numeric_suffixes(string):
         import re
 
-        suffix_str = re.findall("\((.+)\)", string)
+        suffix_str = re.findall(r"\\((.+)\\)", string)
```

Comment:
I think you want `r"\"` or `"\\"`, but not both.

```suggestion
        suffix_str = re.findall(r"\((.+)\)", string)
```

Unless you're specifically trying to match two `\` characters?